relay/api/resources.py

```diff
@@ -349,6 +349,59 @@ def post(self, args, network_address: str):
                 'fees': cost}
 
 
+class ReduceDebtPath(Resource):
+
+    def __init__(self, trustlines: TrustlinesRelay) -> None:
+        self.trustlines = trustlines
+
+    args = {
+        'value': fields.Int(required=False, missing=1),
```

Comment:
Contradicts with the documentation. 

Basically do you think it make sense to just ask for a reduce path for not a specific value? Then it should be optional like it is here.

Reply:
Alright, then there is the same contradiction in the endpoint for Path (/networks/:networkAddress/path-info) where in the documentation it says the "value" field is required, but sets it to "1" when missing in the code. I personally do not think it makes sense but kept it similar for consistency.